modules/internal/DefaultRectangular.chpl

```diff
@@ -21,7 +21,7 @@
 //
 module DefaultRectangular {
   config const dataParTasksPerLocale = 0;
-  config const dataParIgnoreRunningTasks = if CHPL_LOCALE_MODEL=="numa" then true
+  config const dataParIgnoreRunningTasks = if CHPL_LOCALE_MODEL!="flat" then true
```

Comment:
Huh, we should revisit this logic. Doesn't have to be part of this commit, but with this set to false, we can end up heavily oversubscribing the system. See 720c769658358e406745ed8dc11160dd9e61e023 for more info. 

Maybe make an issue to track post-commit TODOs?